### PR TITLE
Add tests for nodetool get/settimeout commands

### PR DESCRIPTION
**Pull request by thobbs**
Done for CASSANDRA-10953

### CODE REVIEW

File: nodetool_test.py [hunk 1]
```diff
@@ -57,3 +57,35 @@ def test_correct_dc_rack_in_nodetool_info(self):
...
+        for timeout_type in types:
+            out, err = node.nodetool('gettimeout %s' % timeout_type)
+            self.assertEqual(0, len(err), err)
+            debug(out)
+            self.assertRegexpMatches(out, r'.* \d+ ms')
+
+        # set all of the timeouts to 123
+        for timeout_type in types:
+            _, err = node.nodetool('settimeout %s 123' % timeout_type)
+            self.assertEqual(0, len(err), err)
```
Comment (ptnapoleon) [inline nodetool_test.py]: This won't work in OSX. Nodetool always prints to stderr there.

File: nodetool_test.py [hunk 2]
```diff
@@ -57,3 +57,35 @@ def test_correct_dc_rack_in_nodetool_info(self):
...
+
+        # verify that they're all reported as 123
+        for timeout_type in types:
+            out, err = node.nodetool('gettimeout %s' % timeout_type)
```
Comment (ptnapoleon) [inline nodetool_test.py]: For this and the other string formatting, can you use the form `'gettimeout {}'.format(timeout_type)`

File: nodetool_test.py (hunk 1 above)
Comment (thobbs) [inline nodetool_test.py]: Hmm, the test above this has the same logic.

File: nodetool_test.py (hunk 2 above)
Comment (thobbs) [inline nodetool_test.py]: This should be fixed now.

**Comment by ptnapoleon** [comment]
+1, then. We can deal with it not passing locally on OSX when we deal with the other test. (I thought I had fixed all of these).

**Pull request merged**